packages/neo-one-node-http-rpc/src/rpcServer$.ts

```diff
@@ -70,6 +87,18 @@ export const rpcServer$ = ({
       const rpcMiddleware = rpc({ blockchain, node });
       router.use(context({ monitor }));
 
+      if (rateLimitOptions !== undefined) {
+        const rateLimitMiddleware = rateLimit(rateLimitOptions);
+
+        router.use(rateLimitMiddleware);
```

Comment:
Just `router.use(rateLimit(rateLimitOptions))`

Reply:
cool, I was thinking similar but wasn't sure if we wanted to always be using it.